reference/src/representation/structs-and-tuples.md

```diff
@@ -0,0 +1,258 @@
+# Representation of structs and tuples
+
+**Disclaimer:** This chapter represents the consensus from issues
+[#11] and [#12].  The statements in here are not (yet) "guaranteed"
+not to change.
+
+[#11]: https://github.com/rust-rfcs/unsafe-code-guidelines/issues/11
+[#12]: https://github.com/rust-rfcs/unsafe-code-guidelines/issues/12
+
+## Tuple types
+
+In general, an anonymous tuple type `(T1..Tn)` of arity N is laid out
+"as if" there were a corresponding tuple struct declared in libcore:
+
+```rust
+#[repr(Rust)]
+struct TupleN<P1..Pn>(P1..Pn);
```

Comment:
> Could add Pn: ?Sized

Does this work already? If not, I'd prefer to open an issue so that we don't forget about this particular extension (allowing the last element of a struct / tuple / ... to be unsized), but to leave it out from the document for now.